src/pages/Test/Test.tsx

```diff
@@ -12,17 +14,31 @@ const Input = styled('input')({
   display: 'none',
 });
 
+const createResourceData: CreateResourceData = {
+  name: 'A113',
+  description: 'This is an intriguing room.',
+  accessRights: ['Admin', 'Teacher'],
```

Comment:
Create constants for `Roles` like I did for severity in `src/consts/constants.ts`